swarm/storage/chunker_test.go

```diff
@@ -444,40 +331,37 @@ func benchmarkSplitPyramidSHA3(n int, t *testing.B) {
 func benchmarkSplitPyramidBMT(n int, t *testing.B) {
 	t.ReportAllocs()
 	for i := 0; i < t.N; i++ {
-		cp := NewChunkerParams()
-		cp.Hash = BMTHash
-		splitter := NewPyramidChunker(cp)
-		tester := &chunkerTester{t: t}
 		data := testDataReader(n)
-		_, _, err := tester.Split(splitter, data, int64(n), nil, nil)
+		putGetter := newTestHasherStore(nil, BMTHash)
+
+		_, _, err := TreeSplit(data, int64(n), putGetter)
 		if err != nil {
-			tester.t.Fatalf(err.Error())
+			t.Fatalf(err.Error())
 		}
 	}
 }
 
 func benchmarkAppendPyramid(n, m int, t *testing.B) {
 	t.ReportAllocs()
 	for i := 0; i < t.N; i++ {
-		chunker := NewPyramidChunker(NewChunkerParams())
-		tester := &chunkerTester{t: t}
 		data := testDataReader(n)
 		data1 := testDataReader(m)
 
-		chunkC := make(chan *Chunk, 1000)
-		key, wait, err := tester.Split(chunker, data, int64(n), chunkC, nil)
+		chunkStore := newTestChunkStore()
+		putGetter := newTestHasherStore(chunkStore, SHA3Hash)
+
+		key, wait, err := PyramidSplit(data, putGetter, putGetter)
 		if err != nil {
-			tester.t.Fatalf(err.Error())
+			t.Fatalf(err.Error())
 		}
 		wait()
-		chunkC = make(chan *Chunk, 1000)
 
-		_, wait, err = tester.Append(chunker, key, data1, chunkC, nil)
+		putGetter = newTestHasherStore(chunkStore, SHA3Hash)
+		_, wait, err = PyramidAppend(key, data1, putGetter, putGetter)
```

Comment:
@nolash Your comment is valid, but since this is really not the scope of my PR to refactor the benchmark tests, I just renamed the function names, see c59a6a477bc8cd21fbbd79c96f60396cb30258dc
Is it better this way?